csrc/codegen.cpp

```diff
@@ -168,6 +168,7 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
     codegen.endBlock();
     NVF_CHECK(codegen.block_nest_level_ == 0);
     std::stringstream final_code;
+    final_code << "// Codegen generated utilities\n";
```

Comment:
How about:

```suggestion
    if (!codegen.utilities_.empty()) {
      final_code << "// Codegen generated utilities\n";
    }
```

Reply:
The reason for always generating this comment is because we are using it to find where the kernel is in `csrc/runtime/compiled_kernel.cpp`. How about changing it to `Codegen generated code`?

Reply:
Sounds good!

Reply:
changed